saleor/graphql/order/dataloaders.py

```diff
@@ -0,0 +1,21 @@
+from collections import defaultdict
+
+from ...warehouse.models import Allocation
+from ..core.dataloaders import DataLoader
+
+
+class AllocationsByOrderLineIdLoader(DataLoader):
+    context_key = "allocations_by_orderline_id"
+
+    def batch_load(self, keys):
```

Comment:
@fowczarek @maarcingebala please take a look at this DataLoader

Reply:
Looks good 👍